pom.xml

```diff
@@ -496,6 +496,8 @@
               <arg>-Xdoclint:all</arg>
             </compilerArgs>
             <encoding>UTF-8</encoding>
+          	<useIncrementalCompilation>true</useIncrementalCompilation>
```

Comment:
It is default value of `maven-compiler-plugin` 
I don't recommend adding more options which is already set. It is no needed complication of configuration.